packages/help-center/src/components/help-center-launchpad.tsx

```diff
@@ -43,6 +44,11 @@ export const HelpCenterLaunchpad = () => {
 		siteSlug = window?.location?.host;
 	}
 
+	const { data } = useLaunchpadChecklist( siteSlug, siteIntent || '' );
+	const totalLaunchpadSteps = data?.checklist?.length || 4;
```

Comment:
Curious why we're providing a default value of `4` here? I assume it's to avoid flashing styles when the data is still being fetched?

Reply:
@Sharpirate - I kept the same default values that had been hard coded here before this PR (1 out of 4 steps, or 25%). So if we don't get reliable complete / total steps data, we'd just render the icon the same as we did in the past.